docs/operations/post_process.md

```diff
@@ -0,0 +1,154 @@
+
+# post_process #
+
+This processor encapsulates the post_process phase of [ts-transforms](https://terascope.github.io/teraslice/docs/packages/ts-transforms/overview). This is meant to work in conjunction with the selection processor as this works by using the metadata set in the selection processor.
+
+
+
+
+| Configuration | Description | Type |  Notes |
+| --------- | -------- | ------ | ------ |
+| _op | Name of operation, it must reflect the exact name of the file | String | required |
+| rules | an array of strings that are the locations where rule files. must be specifed in "assetName:path" format | String[] | required |
+| plugins | an array of strings that are the locations where [plugins](https://terascope.github.io/teraslice/docs/packages/ts-transforms/plugins) reside. must be specifed in "assetName:modulePath" format | Object[] | optional, defaults to [] |
+| type_config | a schema for the data being consumed. Set the keys to your data's field names, with values set to this [enum](https://terascope.github.io/teraslice/docs/packages/types/api/enums/xlucenefieldtype) | Object | optional |
+| variables | An object containing any varialbes for the xlucene rules | Object | optional|
+
+
+## Usage
+
+```typescript
+
+// someAssetId:transformRules.txt
```

Comment:
This should be broken into multiple code blocks for each file so it is easier to read